packages/components/src/components/post-megadropdown-toggle/post-megadropdown-toggle.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, Event, EventEmitter, Host, State, h } from '@stencil/core';
+
+@Component({
+  tag: 'post-megadropdown-toggle',
+  shadow: true,
+  styleUrl: './post-megadropdown-toggle.scss',
+})
+export class PostMegadropdownToggle {
+  @Event() postMegadropdownToggled: EventEmitter;
+
+  @State() toggled = false;
```

Comment:
Is this state necessary? It does not seem to be used anywhere.

Reply:
Added usage for `aria-expanded`.